packages/fluentui/docs/src/examples/components/Ref/Types/RefForwardingExample.shorthand.tsx

```diff
@@ -5,7 +5,9 @@ type RefForwardingExampleState = {
   isMounted: boolean;
 };
 
-const ExampleButton = React.forwardRef<HTMLButtonElement, { children: React.ReactNode }>((props, ref) => (
+const ExampleButton: React.FunctionComponent<{ children: React.ReactNode } & React.RefAttributes<
```

Comment:
was this change necessary ? I managed to build the docs package without this change without any issues, and vscode doesn't seem to be screaming either

Reply:
Good question. Not sure why this change was made, but I've reverted it.